torch/utils/_stats.py

```diff
@@ -3,8 +3,13 @@
 # AND SCRUB AWAY TORCH NOTIONS THERE.
 import collections
 import functools
+from typing import OrderedDict
 
-simple_call_counter = collections.OrderedDict()
+simple_call_counter: OrderedDict[str, int] = collections.OrderedDict()
```

Comment:
why do we need OrderedDict ? isn't dict already ordered ?

Reply:
ask @voznesenskym

Reply:
I like preserving key order for things like this.